lib/scala/project-manager/src/main/resources/META-INF/native-image/org/enso/projectmanager/resource-config.json

```diff
@@ -1,77 +1,127 @@
-    },
-    {
-      "pattern": "\\Qorg/enso/projectmanager/service/versionmanagement/RuntimeVersionManagerMixin$ErrorRecovery.class\\E"
-    },
-    { "pattern": "\\Qorg/slf4j/impl/StaticLoggerBinder.class\\E" },
-    { "pattern": "\\QMETA-INF/services/org.enso.logger.LoggerSetup\\E" },
-    { "pattern": "\\QMETA-INF/services/org.enso.logging.LogbackLoggingServiceFactory\\E" },
-    { "pattern": "\\Qch/qos/logback/classic/spi/Configurator.class\\E" },
-    { "pattern": "\\Qreference.conf\\E" },
-    { "pattern": "\\Qversion.conf\\E" },
-    { "pattern": "\\Qzio/App$$anon$1.class\\E" },
-    { "pattern": "\\Qzio/App.class\\E" },
-    { "pattern": "\\Qzio/Fiber.class\\E" },
-    { "pattern": "\\Qzio/ZIO$.class\\E" },
-    { "pattern": "\\Qzio/ZIO$TimeoutTo.class\\E" },
-    { "pattern": "\\Qzio/ZIO$ZipLeftFn.class\\E" },
-    { "pattern": "\\Qzio/ZIO.class\\E" },
-    { "pattern": "\\Qzio/ZScope$.class\\E" },
-    { "pattern": "\\Qzio/blocking/package$.class\\E" },
-    { "pattern": "\\Qzio/clock/PlatformSpecific$$anon$1.class\\E" },
-    { "pattern": "\\Qzio/clock/package$.class\\E" },
-    { "pattern": "\\Qzio/clock/package$Clock$Service$$anon$1.class\\E" },
-    { "pattern": "\\Qzio/console/package$.class\\E" },
-    { "pattern": "\\Qzio/console/package$Console$Service$$anon$1.class\\E" },
-    { "pattern": "\\Qzio/internal/FiberContext$InterruptExit$.class\\E" },
-    { "pattern": "\\Qzio/internal/FiberContext.class\\E" }
-  ],
-  "bundles": []
+  "resources":{
+  "includes":[{
+    "pattern":"\\QMETA-INF/MANIFEST.MF\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/ch.qos.logback.classic.spi.Configurator\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.lang.System$LoggerFinder\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.net.spi.InetAddressResolverProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.net.spi.URLStreamHandlerProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.nio.channels.spi.SelectorProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.nio.charset.spi.CharsetProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.time.zone.ZoneRulesProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.enso.logger.LoggerSetup\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.enso.logging.LogbackLoggingServiceFactory\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.enso.logging.LoggingServiceFactory\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.slf4j.spi.SLF4JServiceProvider\\E"
+  }, {
+    "pattern":"\\Qakka-http-version.conf\\E"
+  }, {
+    "pattern":"\\Qakka-http-version.json\\E"
+  }, {
+    "pattern":"\\Qakka-http-version.properties\\E"
+  }, {
+    "pattern":"\\Qapplication.conf\\E"
+  }, {
+    "pattern":"\\Qapplication.json\\E"
+  }, {
+    "pattern":"\\Qapplication.properties\\E"
+  }, {
+    "pattern":"\\Qch/qos/logback/classic/spi/Configurator.class\\E"
+  }, {
+    "pattern":"\\Qcom/sun/jna/win32-x86-64/jnidispatch.dll\\E"
```

Comment:
Oh, so we are really putting a `.dll` and `.so` & co. libraries into the binary, they get extracted to `/tmp` and used later. Horrible.